src/crimson/os/seastore/seastore.cc

```diff
@@ -896,9 +896,8 @@ SeaStore::_omap_list_ret SeaStore::_omap_list(
   return seastar::do_with(
     BtreeOMapManager(*transaction_manager),
     root,
-    start,
-    [&t, config](auto &manager, auto& root, auto& start) {
-      return manager.omap_list(root, t, start, config);
+    [&t, start, config](auto &manager, auto& root) {
```

Comment:
Same here, both `start` and `end` to `omap_list()` need to be wrapped with do_with() by caller according to the interface description.